compose/hubmap.stage.yml

```diff
@@ -20,9 +20,9 @@ services:
     networks:
       - gateway_hubmap
 
-  portal-ui-prod:
-    hostname: portal-ui-prod
-    container_name: portal-ui-prod
+  prod-portal-ui:
```

Comment:
```suggestion
  # Service names need to be kept in sync with gateway:
  # https://github.com/hubmapconsortium/gateway/blob/f2c2d5d/nginx/conf.d-stage/prod-portal-ui.conf
  # Some part of the routing seems to be be based on left substrings, rather than names, so make sure left substrings are unique.
  prod-portal-ui:
```